portfolio/src/main/webapp/script.js

```diff
@@ -48,19 +48,20 @@ async function fetchContent() {
   const messagesEl = $('#comments-container');
   messagesEl.empty();
   for (message of messages) {
-    messagesEl.append(createListElement(message));
+    messagesEl.append(createListElement(message.author, message.text));
   }
 }
 
 /**
  * Creates a <div> element containing author and comment.
- * @param {string} text The text to create list element with
+ * @param {string} author The author of the comment
+ * @param {string} text The text of the comment
  * @return {Element} a div element
  */
-function createListElement(text) {
-  return $(html`
```

Comment:
did you mean to remove this? keep it if Prettier works better with it.

Reply:
I got prettier to format it already, so it's no longer needed. When I kept it, I got a bunch of errors in the js console.